ci/debian/control

```diff
@@ -27,15 +27,18 @@ Description: Centreon Clib is a common library for all Centreon products
 
 Package: centreon-clib-dev
 Architecture: any
-Depends: centreon-clib, ${misc:Depends}, ${shlibs:Depends}
+Depends: 
+    centreon-clib (>= 22.04.0~),
```

Comment:
it would be better to set dynamicaly this variable

Reply:
> it would be better to set dynamicaly this variable

Yes, It will really make maintenance easier. I will implement this and other packages.
Thanks @rem31 !